docs/book/security.md

```diff
@@ -25,7 +25,11 @@ is specified.
 
 By default, OPA ignores insecure HTTP connections when TLS is enabled. To allow
 insecure HTTP connections in addition to HTTPS connections, provide another
-listening address with `--insecure-addr`.
+listening address with `--addr`. For example:
+
+```bash
+opa run --server --log-level debug --tls-cert-file public.crt --tls-private-key-file private.key --addr https://0.0.0.0:8181 --addr http://localhost:8282
```

Comment:
It might be more readable if this was split over a few lines. For example:

```
opa run --server \
  --log-level debug \
  --tls-cert-file public.crt \
  --tls-private-key-file private.key \
  --addr https://0.0.0.0:8181 \
  --addr http://localhost:8282
```